packages/repack/ios/ScriptConfig.h

```diff
@@ -17,16 +17,16 @@
 + (nonnull ScriptConfig *)fromConfigDictionary:(nonnull NSDictionary *)config
                                   withScriptId:(nonnull NSString*)scriptId;
 
-- (ScriptConfig *)initWithScript:(nonnull NSString*)scriptId
-                       withURL:(nonnull NSURL*)url
-                    withMethod:(nonnull NSString*)method
-                     withQuery:(nullable NSString*)query
-                     withFetch:(BOOL)fetch
-                  withAbsolute:(BOOL)absolute
-                   withHeaders:(nullable NSDictionary*)headers
-                      withBody:(nullable NSData*)body
-                   withTimeout:(nonnull NSNumber*)timeout
-     withVerifyScriptSignature:(nonnull NSString*)verifyScriptSignature;
+- (nonnull ScriptConfig *)initWithScript:(nonnull NSString*)scriptId
+                         withURL:(nonnull NSURL*)url
+                      withMethod:(nonnull NSString*)method
+                       withQuery:(nullable NSString*)query
+                       withFetch:(BOOL)fetch
+                    withAbsolute:(BOOL)absolute
+                     withHeaders:(nullable NSDictionary*)headers
+                        withBody:(nullable NSData*)body
+                     withTimeout:(nonnull NSNumber*)timeout
+       withVerifyScriptSignature:(nonnull NSString*)verifyScriptSignature;
```

Comment:
You can actually only specify what's nullable by using `NS_ASSUME_NONNULL_BEGIN` (where it assumes everything is nonnull by default) and you only specify nullable stuff. I thinki having all of these annotations makes the code less readable. 

Here is a reference: https://developer.apple.com/documentation/swift/designating-nullability-in-objective-c-apis